src/components/Request/FormFields/AdvertisementLength.tsx

```diff
@@ -1,19 +1,31 @@
-import { Radio } from "@fluentui/react-components";
 import "components/Request/Request.css";
 import { RHFRequest } from "components/Request/NewRequestForm";
-import BACRadioGroup from "components/BaseFormFields/BACRadioGroup";
+import BACInput from "components/BaseFormFields/BACInput";
 
 const AdvertisementLength = () => {
   return (
     <div className="requestFieldContainer">
-      <BACRadioGroup<RHFRequest>
+      <BACInput<RHFRequest>
         name="advertisementLength"
         labelText="Advertisement Length"
-        fieldProps={{ layout: "horizontal" }}
-      >
-        <Radio value="Normal" label="Normal Period" />
-        <Radio value="Extended" label="Extended Period (> 14 days)" />
-      </BACRadioGroup>
+        rules={{
+          required: "Close date is required",
```

Comment:
Didn't see this earlier. Should probably say "Advertisement Length is required".